packages/mobile/package.json

```diff
@@ -15,6 +15,7 @@
         "storybook": "start-storybook -p 6006"
     },
     "dependencies": {
+        "@expo/match-media": "^0.0.0-beta.2",
```

Comment:
I'm wondering if this still need to be there considering the Entry component is no longer tests. It might be worth a check.
Same as the one below regarding pinning.